ui/widgets/accordion.js

```diff
@@ -78,6 +78,9 @@ return $.widget( "ui.accordion", {
 		var options = this.options;
 
 		this.prevShow = this.prevHide = $();
+		if ( this.isRtl() ) {
+			this._addClass( ".ui-accordion-rtl" );
```

Comment:
This shouldn't have a leading dot. I'm also not sure it should be a keyed class.